index.js

```diff
@@ -9,38 +13,42 @@ class DuplicateCompatError extends Error {
   }
 }
 
-function load() {
-  // Recursively load one or more directories passed as arguments.
+/**
+ * Recursively load one or more directories passed as arguments.
+ *
+ * @param {string[]} dirs The directories to load
+ * @returns {object} All of the browser compatibility data
+ */
+function load(...dirs) {
   let dir,
     result = {};
 
-  function processFilename(fn) {
-    const fp = path.join(dir, fn);
-    let extra;
+  for (dir of dirs) {
+    dir = path.isAbsolute(dir) ? dir : path.resolve(__dirname, dir);
```

Comment:
I don't think this really helps my concern, `path.resolve()` probably has an "is absolute" check internally, so this is redundant / mistrusting of `path.resolve`.

How about we put the `path.resolve()` when resolving the full path instead?